app/views/application/dashboard.html.erb

```diff
@@ -27,10 +27,18 @@
               <% if user_signed_in? && cat.new_posts_for?(current_user) %>
                 <span class="badge is-status" title="unread activity in category"></span>
               <% end %>
-            <% end %>
+	      <% if current_user&.has_ability_on(c.id, 'edit_posts') %>
+	      <% sug_edits = SuggestedEdit.where(post: Post.undeleted.where(category: cat), active: true).count %>
+	      <% if sug_edits != 0 %>
+	      &nbsp;&nbsp;(<%= sug_edits %> pending edits)

```

Comment:
I admit this is ugly.  What's the _proper_ way to get some breathing room around the new-posts badge before diving into the edits text?